src/it/README.md

```diff
@@ -0,0 +1,36 @@
+# Sonar Scanner Maven Plugin Integration Tests
```

Comment:
I think we already have some documentation about ITs here: https://github.com/SonarSource/sonar-scanner-maven/blob/master/develop.md

What do you think if we merge these two documents, or move all the information about ITs here?

Reply:
Oh nice! I will remove it then :)